scripts/redshift/transformers/cell_expression.py

```diff
@@ -163,21 +169,17 @@ def _parse_keys(self, bundle_dir):
         ap_path = p.joinpath("analysis_protocol_0.json")
         ap_key = json.load(open(ap_path))["provenance"]["document_id"]
 
-        donor_paths = list(p.glob("donor_organism_*.json"))
-        if len(donor_paths) > 1:
-            donor_key = ""
-        else:
-            donor_key = json.load(open(donor_paths[0]))["provenance"]["document_id"]
+        specimen_paths = list(p.glob("specimen_from_organism_*.json"))
+        specimen_keys = [json.load(open(p))["provenance"]["document_id"] for p in specimen_paths]
+        specimen_key = sorted(specimen_keys)[0]
```

Comment:
Can you explain why we can ignore the other keys?

Reply:
So there's usually only one specimen. But in cases where there's more than one, we want to report as much as we can. For example, there are a bunch of bundles currently in the prod DSS that have four donor organisms and four specimens that end up in a single cell suspension. Each of those donors is different, different ages, sex, etc, but they share a bunch of attributes like genus or disease. The strategy here is to just pick one of the ids and attach all the common attributes to it. There's a similar `sorted(...)[0]` in `SpecimenLibraryTransformer._parse_keys`.

Reply:
I see. So it's unlikely (or not possible) for a single specimen to show up in another bundle with a different set of specimens? i.e. this `sorted(...)[0]` should uniquely identify a set of specimens/donors?

Reply:
Hmmm yeah that's a good question. I don't think that has happened yet, but I'm assuming it could.